lib/client.js

```diff
@@ -49,11 +51,7 @@ Client.prototype.report = function report (message, callback) {
     callback = noop;
   }
 
-  this._client.request({
-    method: this._reporting.method,
-    path: this._reporting.path,
-    payload: message
-  }, callback);
+  this._client.request({ path: 'report', payload: message }, callback);
```

Comment:
I'm fine with using the route id over method and path. I like the idea of learning it from the server because the server routes can be more easily changed, without requiring changes to the client. I was thinking of it kinda like HATEOAS over WebSockets.